Dockerfile

```diff
@@ -27,17 +25,18 @@ RUN pixi install -e pkg
 # Download the punkt NLTK data
 RUN pixi run -e production python -m nltk.downloader -d build/nltk_data punkt
 # Install poprox-recommender
-RUN pixi run -e production pip install --no-deps .
+RUN pixi run -e production pip install --no-deps --root-user-action=ignore .
 # Pack up the environment for migration to runtime
 RUN ./.pixi/envs/pkg/bin/conda-pack -p .pixi/envs/production -d /opt/poprox -o build/production-env.tar
 
 # Use Lambda "Provided" base image for the deployment container
 # We installed Python ourselves
 FROM public.ecr.aws/lambda/provided:al2023
+ARG LOG_LEVEL=INFO
 
 # Unpack the packaged environment from build container into runtime contianer
 # GNU tar chokes on conda-pack's output, so we use bsdtar
-RUN dnf install -y bsdtar
+RUN dnf install -y bsdtar && dnf clean -y all
```

Comment:
This is cleaning up the DNF caches, saving about 60Mb in the final image. I don't do it in the build image since we aren't deploying that image.